server/domain/academic-record/models/AcademicRecord.ts

```
@@ -0,0 +1,182 @@
+
+  getStudentId(): string | null {
+    return this.studentId;
+  }
+
+  getSemesters(): SemesterGrade[] {
+    if (!this.semesters) {
+      return [];
+    }
+
+    return [...this.semesters].sort((a, b) => {
+      if (b.getYear() !== a.getYear()) return b.getYear() - a.getYear();
```

ESLint

[error] Expected { after 'if' condition.

(curly)